packages/citizen-sdk/README-ClaimSDK.md

```diff
@@ -0,0 +1,228 @@
+# Additions to README for Claim SDK
+
+the `ClaimSDK` facilitates claiming Universal Basic Income (UBI) through the GoodDollar Protocol.
+
+## Getting Started
+
+### Using the Wagmi SDK for Claim
+
+To integrate the Claim SDK with Wagmi in a React application, you need to obtain the necessary blockchain clients and an instance of the Identity SDK, as the Claim SDK depends on it. Below is an example of how to initialize the Claim SDK within a React component.
+
+```typescript
+import { useAccount, usePublicClient, useWalletClient } from 'wagmi';
+import { useIdentitySDK } from '@goodsdks/identity-sdk/wagmi-sdk';
+import { ClaimSDK } from '@goodsdks/identity-sdk/viem-claim-sdk';
+
+const ClaimComponent = () => {
+  const { address } = useAccount();
+  const publicClient = usePublicClient();
+  const { data: walletClient } = useWalletClient();
+  const identitySDK = useIdentitySDK('production');
+
+  if (!address || !publicClient || !walletClient || !identitySDK) {
+    return <div>Loading...</div>;
+  }
+
+  const claimSDK = new ClaimSDK({
+    account: address,
+    publicClient,
+    walletClient,
+    identitySDK,
+    env: 'production',
+  });
+
+  // Use claimSDK to call methods like claim(), checkEntitlement(), etc.
+  return <div>Claim SDK Initialized</div>;
+};
+```
+
+### Using the Viem SDK for Claim
+
+For non-React environments or backend services, you can initialize the Claim SDK directly using Viem's clients. The `ClaimSDK.init` method simplifies the process by fetching the account address from the wallet client.
+
+```typescript
+import { PublicClient, WalletClient } from "viem"
+import { IdentitySDK } from "@goodsdks/identity-sdk/viem-identity-sdk"
+import { ClaimSDK } from "@goodsdks/identity-sdk/viem-claim-sdk"
+
+const publicClient = new PublicClient({
```

Comment:
refer to the full demo code how it is done?

Reply:
done